shipyard/src/main/java/org/jclouds/shipyard/features/ContainersApi.java

```diff
@@ -64,6 +66,7 @@
    void restartContainer(@PathParam("id") String id);
    
    @Named("containers:deploy")
+   @Fallback(NullOnContainerResourceUnavailableAnd500.class)
```

Comment:
Hmmm does it just return `[ { object } ]`, or does it return something like:
```json
{ "foo": [ { "id": "foo", "bar": "bar" } ] }
```
I mean, the returned list is inside a json object or does it directly return the raw list?